pykern/pkunit.py

```diff
@@ -669,6 +677,7 @@ def _validate_args(self, expect_path, *args, **kwargs):
         self._expect_is_jinja = self._expect_path.ext == ".jinja"
         self._is_ndiff = self._expect_path.ext == ".ndiff"
         self._ndiff_epsilon = kwargs.get("ndiff_epsilon", 1e-13)
```

Comment:
When changing the meaning of a parameter, you should maybe make a new parameter so call it ndiff_rel_epsilon. Then, in the code, you throw an error if someone passes ndiff_epsilon with a message about what's changed. You can deprecate, but since it's little used, it's probably not important.

This is a little used interface, but it's the principle that's important here. 

If is_ndiff, then rel or abs should be passed, but not both. You can choose an appropriate default (rel, 1e-13), but it should be after checking the parameters aren't conflicting.

Reply:
What do you think about passing just epsilon and a kwarg use_absolute for if someone wants to use absolute difference? Then there is only one epsilon.

Reply:
`ndiff_epsilon` and `ndiff_epsilon_is_abs`.